notebooks/search/03-ELSER.ipynb

```diff
@@ -57,16 +57,17 @@
    },
    "source": [
     "# Setup ELSER\n",
-    "To use ELSER, you must have the [appropriate subscription]() level\n",
+    "\n",
+    "From 8.11, Elastic supports  `.elser_model_2` model which offers an improved retrieval accuracy. To use ELSER, you must have the [appropriate subscription]() level\n",
     "for semantic search or the trial period activated.\n",
     "\n",
-    "Follow these [instructions](https://www.elastic.co/guide/en/machine-learning/8.8/ml-nlp-elser.html#trained-model) to download and deploy ELSER in the Kibana UI or using the Dev Tools **Console**.\n",
+    "Follow these [instructions](https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html#trained-model) to download and deploy ELSER  `.elser_model_2` in the Kibana UI or using the Dev Tools **Console**.\n",
```

Comment:
As stated above, I think we could add support for ELSER v1 as a separate section, or as an additional script, so v1 users can use the notebook

Reply:
Same as above. 

> Based on the outcome of discussion in [thread](https://elastic.slack.com/archives/C05E3NSDLS2/p1697556832684659?thread_ts=1696523296.981769&cid=C05E3NSDLS2), we will be focussing one one notebook for ELSER v2 model.